### PR TITLE
Add data example upload

### LINKED ISSUE #5: Add data example upload
Related to https://github.com/fredrikhgrelland/terraform-nomad-presto/pull/7

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- Closes #5 

## What was added/changed/fixed?
- [added] example data files; playbook
- [updated] doc
- [rm] vagrant_box_example directory

## Related issue(s)? [Optional]
presto data upload [#5](https://github.com/fredrikhgrelland/terraform-nomad-presto/issues/5)

## Checklist (after created PR)
- [X] Added reviewers
- [X] Added assignee(s)
- [X] Added label(s)
- [X] Added to project
- [X] Added to milestone

### CODE REVIEW

**Review by dangernil — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -169,5 +170,23 @@ OK
 +----------------+
 ```
 
+#### Data example upload
+[example/resources/data](example) directory contains data sample to upload. You can create tables via [beeline cli](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93CommandLineShell), how to use beeline-cli see section [verifying setup](#verifying-setup).
```
Comment (claesgill) [inline README.md:174]:
```suggestion
The [example/resources/data](example/resources/data) directory contains a data sample you can upload. 
To upload data and create tables you can use the [beeline cli](https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93CommandLineShell), see the sql queries bellow. However, if you're not familiar with the `beeline-cli`, see the [verifying setup](#verifying-setup) section.
```

Is this correct or is the data already uploaded at this point?

Comment (zhenik) [reply README.md:174]: The data files have been already uploaded at this point. All that user need to do is to create the table in hive, to verify that

File: dev/ansible/04_upload_files.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - test
+
+# To quickly verify that proxy available (without any credentials)
+# https://github.com/minio/minio/issues/4851#issuecomment-324543717
+- name: Minio curl example. Verify that proxy to minio available
+  uri:
+    url: http://127.0.0.1:9999
+    status_code: 403
+  register: curl_output
+  until: curl_output.status == 403
```
Comment (claesgill) [inline dev/ansible/04_upload_files.yml:33]: Why is the status `403`?

Comment (zhenik) [reply dev/ansible/04_upload_files.yml:33]: lack of intention. Fixed in https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/39/commits/34f1c8341321e50f01bf1707dc188f5c2f9d079d

File: example/resources/data/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# File types
+Different file types which are uploaded to stack in current example
```
Comment (claesgill) [inline example/resources/data/README.md:2]: What is stack and current example?

File: example/resources/data/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## CSV
+
```
Comment (claesgill) [inline example/resources/data/README.md:5]:
```suggestion
```

File: dev/ansible/04_upload_files.yml (hunk 2 above)
Comment (claesgill) [reply dev/ansible/04_upload_files.yml:33]: Ahh, got it! 👍

File: example/resources/data/README.md (hunk 3 above)
Comment (zhenik) [reply example/resources/data/README.md:2]: Sry copy-paste from presto module. I will fix it

**Review by claesgill — APPROVED** [review]
LGTM

**Review by dangernil — APPROVED** [review]

**Pull request merged**